routers/api/actorlist/Actorlist.js

```diff
@@ -1,28 +1,28 @@
-const models=require('../../../models');
-
-async function actorList(req,res,next)
-{
-    try{
+const models = require('../../../models');
+const Logger = require('../../../services/logger')
```

Comment:
Use camel casing. Eg: logger=

Reply:
I have logger already for that Logger is class in services/logger.js that's why i used